### ISSUE TITLE
Unable to install on 8bit InterSystems IRIS

### ISSUE THREAD

**Issue by eduard93** [intersystems/ipm#121]
Load&compile fails with the error

```
ERROR #5001: module.xml is malformed.
ERROR #5001: module.xml is malformed.2020-07-14 19:41:25 0 %ZPM.Installer: ОШИБКА #5001: module.xml is malformed.
2020-07-14 19:41:25 0 %ZPM.Installer: Installation failed at 2020-07-14 19:41:25
2020-07-14 19:41:25 0 %Installer: Elapsed time 5.476291s
Time spent 5.51875
ERROR #5001: module.xml is malformed.
  > ERROR #5090: An error has occurred while creating projection %ZPM.Installer:Reference.
Detected 1 errors during load.
```

On `IRIS for Windows (x86-64) 2020.1 (Build 215) Mon Mar 30 2020 20:30:26 EDT`

**Comment by evshvarov** [comment]
@eduard93 please check

**Comment by isc-tleavitt** [comment]
I'm no longer able to reproduce this, assuming this was connected to the locale-related issues that we fixed in #411
